### PR TITLE
Treadmill programmed by Paul

### PR DESCRIPTION
**Pull request by fabiansinz**

### CODE REVIEW

File: python/pipeline/behavior.py [hunk 1]
```diff
@@ -63,4 +67,66 @@ def _make_tuples(self, key):
...
+        ball_vel = ball_time_to_vel(ball_time)
+
+        #assign calculated properties to key
+        key['treadmill_time'] = ball_time
+        key['treadmill_raw'] = ball_raw
+        key['treadmill_vel'] = ball_vel
+
+        #insert and notify user
+        self.insert1(key)
+        self.notify(key)
```
Comment (ecobost) [inline python/pipeline/behavior.py]: small issue here is that the notification will have treadmill_time, treadmill_raw and treadmill_vel. One could do:
```python
tuple_ = key.copy()
tuple_['treadmill_time'] = treadmill_time
...
self.insert1(tuple_)
self.notify(key)
```

File: python/pipeline/utils/eye_tracking.py [hunk 2]
```diff
@@ -261,13 +261,24 @@ def get_pupil_from_contours(self, contours, small_gray, show_matching=5):
...
 
         small_gray = gray[slice(*eye_roi[0]), slice(*eye_roi[1])]
-        blur = cv2.GaussianBlur(small_gray, (h, h), 0)
+        c = 0.05
+        p = 7
+        if self._running_avg is None:
+            self._running_avg = np.array(small_gray) ** p
+        else:
+            self._running_avg = c * np.array(small_gray) ** p + (1 - c) * self._running_avg
+            small_gray += self._running_avg.astype(np.uint8) - small_gray # big hack
```
[the remark this answers is not in the document]
Comment (fabiansinz) [reply python/pipeline/utils/eye_tracking.py]: That's just a hack currently to write the content of the ROI in the bigger image that's shown when the video is being processed.

**Comment by ecobost** [comment]
@fabiansinz @paulgfahey Should I review this PR or are you still working on it?

**Comment by jiakunf** [comment]
We'll fix the schema name. It's too late for the data that's in there already.

**Pull request merged**